contributed_definitions/nyaml/NXsensor_scan.yaml

```diff
@@ -1,4 +1,4 @@
-category: application
+category: base
```

Comment:
this is an AppDef for a generic scanning experiment using NXsensors

Reply:
My bad, it should not be a base class. back to `application`.